automation_tools/__init__.py

```diff
@@ -7,11 +7,13 @@
 from __future__ import print_function
 import os
 import random
+import requests
 import socket
 import sys
 import time
 import novaclient
 import subprocess
+from lxml import html
```

Comment:
I think we can use some xml parsing module from stdlib and avoid adding another dependency.